analytics_data_api/v0/serializers.py

```diff
@@ -45,6 +46,40 @@ class ModelSerializerWithCreatedField(serializers.ModelSerializer):
     created = serializers.DateTimeField(format=settings.DATETIME_FORMAT)
 
 
+class DynamicFieldsSerializerMixin(object):
+    """
+    Allows the `fields` query parameter to determine which fields should be returned in the response.
+    """
+    fields_sep = ','
+
+    def get_fields(self):
+        """
+        Filter the list of available fields based on the list of fields passed to the request.
+        """
+        fields = super(DynamicFieldsSerializerMixin, self).get_fields()
+
+        request = self.context.get('request')
+        if request:
+            request_fields = request.QUERY_PARAMS.get('fields')
+            if request_fields:
```

Comment:
Ah yes.. I have a bad habit of relying on language-specific `False`.  Thanks for catching that!

Addressed with [f5b2ce2](https://github.com/edx/edx-analytics-data-api/pull/128/commits/f5b2ce2b7da0aeb684f110699d7cbb4866f2ea3e).